<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogForm</class>
 <widget class="QWidget" name="LogForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recourse.qrc">
    <normaloff>:/icon1.png</normaloff>:/icon1.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(26,148,188);
border:none
	
	





</string>
  </property>
  <widget class="QGroupBox" name="LogGB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(20, 196, 188);
border-radius:30px;
background-color:rgb(255,255,255);

</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="LogStrL">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>40</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 290 20pt &quot;Microsoft YaHei UI&quot;;
border:none</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="IdT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>361</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border:2px solid rgb(20,196,188);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输入Id</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PswdT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>361</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border:2px solid rgb(20,196,188);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MbrRegB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MbrLogB">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MbrRb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">back-ground-color:rgb(255,255,255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;
border:none</string>
    </property>
    <property name="text">
     <string>会员</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SysRb">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>350</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">back-ground-color:rgb(255,255,255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;
border:none</string>
    </property>
    <property name="text">
     <string>系统管理员</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MgrRb">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>350</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">back-ground-color:rgb(255,255,255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;
border:none</string>
    </property>
    <property name="text">
     <string>场地管理员</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SysMgrLogB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="LogInfoL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Microsoft YaHei UI&quot;;
color:rgb(204,22,58);
border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="BottomGB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>640</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:none</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MinB">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
		background-color:rgb(21, 131, 161);
}
QPushButton:pressed{
	background-color:rgb(17, 111, 134);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="recourse.qrc">
     <normaloff>:/icon3.png</normaloff>:/icon3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>19</width>
     <height>19</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
		background-color: rgb(255, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(162, 0, 3);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="recourse.qrc">
     <normaloff>:/icon2.png</normaloff>:/icon2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <zorder>BottomGB</zorder>
  <zorder>LogGB</zorder>
  <zorder>MinB</zorder>
  <zorder>CloseB</zorder>
 </widget>
 <resources>
  <include location="recourse.qrc"/>
 </resources>
 <connections/>
</ui>
